src/node/index.ts

```diff
@@ -272,9 +273,9 @@ export default function PluginInspect(options: Options = {}): Plugin {
       const result = typeof _result === 'string' ? _result : _result?.code
       const sourcemaps = typeof _result === 'string' ? null : _result?.map
       const map = ssr ? transformMapSSR : transformMap
-      if (filter(id) && result != null) {
+      if (filter(id)) {
```

Comment:
I don't understand why we should record them? Returning `undefined` should indicate the plugin is bailed out.

Reply:
I need to find out if a plugin has executed even if it bailed out. Also, figure out the plugin's order.

Reply:
Ok, I guess then for `result == null`, the label should better to be different from `no changes`, but maybe `bailout`?

Reply:
Done